src/sentry/release_health/metrics.py

```diff
@@ -458,165 +447,105 @@ def run_sessions_query(
-                ],
-                granularity=Granularity(LEGACY_SESSIONS_DEFAULT_ROLLUP),
-            )
-            request = Request(
-                dataset=Dataset.Metrics.value, app_id=SnubaAppID, query=init_sessions_query
-            )
-            rows = raw_snql_query(
-                request,
-                referrer="release_health.metrics.get_release_sessions_time_bounds.init_sessions",
-                use_cache=False,
-            )["data"]
-        except MetricIndexNotFound:
-            rows = []
-
-        try:
-            # Take care of potential timestamp updates by looking at the metric
-            # for session duration, which is emitted once a session is closed ("terminal state")
-            #
-            # There is a testcase checked in that tests specifically for a
-            # session update that lowers session.started. We don't know if that
-            # testcase matters particularly.
-            terminal_sessions_query = Query(
-                match=Entity(EntityKey.MetricsDistributions.value),
-                select=select,
-                where=where
-                + [
-                    Condition(
-                        Column("metric_id"),
-                        Op.EQ,
-                        resolve(USE_CASE_ID, org_id, SessionMRI.RAW_DURATION.value),
-                    ),
-                ],
-                granularity=Granularity(LEGACY_SESSIONS_DEFAULT_ROLLUP),
             )
-            request = Request(
-                dataset=Dataset.Metrics.value, app_id=SnubaAppID, query=terminal_sessions_query
-            )
-            rows.extend(
-                raw_snql_query(
-                    request,
-                    referrer="release_health.metrics.get_release_sessions_time_bounds.terminal_sessions",
-                    use_cache=False,
-                )["data"]
+
+        if environments:
+            where.append(
+                Condition(
+                    lhs=Column(name="tags[environment]"),
+                    op=Op.IN,
+                    rhs=environments,
+                )
             )
-        except MetricIndexNotFound:
-            pass
-
-        # This check is added because if there are no sessions found, then the
-        # aggregations query return both the sessions_lower_bound and the
-        # sessions_upper_bound as `0` timestamp and we do not want that behaviour
-        # by default
-        # P.S. To avoid confusion the `0` timestamp which is '1970-01-01 00:00:00'
-        # is rendered as '0000-00-00 00:00:00' in clickhouse shell
-        formatted_unix_start_time = datetime.utcfromtimestamp(0).strftime("%Y-%m-%dT%H:%M:%S+00:00")
 
-        lower_bound: Optional[str] = None
-        upper_bound: Optional[str] = None
+        query = MetricsQuery(
+            org_id=org_id,
+            project_ids=[project_id],
+            select=select,
+            start=datetime(2008, 5, 8, tzinfo=timezone.utc),  # Sentry first commit
```

Comment:
```suggestion
SENTRY_FIRST_COMMIT_DATETIME = datetime(2008, 5, 8, tzinfo=timezone.utc)

            start=SENTRY_FIRST_COMMIT_DATETIME,
```

Reply:
Yes, it's a good idea ( I thought about putting it at the top of the file, but then I left it as it was in the original code).
I'll change it to your suggestion.